src/views/templates/details/tabs/details/TemplateDetailsPage.tsx

```diff
@@ -48,7 +49,7 @@ const TemplateDetailsPage: React.FC<TemplateDetailsPageProps> = ({ obj: template
             <SidebarEditorSwitch />
             <Grid className="margin-top">
               <GridItem span={5} className="margin-top-grid-item">
-                <TemplateDetailsLeftGrid template={resource} editable={!isCommonTemplate} />
+                <TemplateDetailsLeftGrid template={resource} editable={isTemplateEditable} />
```

Comment:
template already passed to TemplateDetailsLeftGrid as resource , you can simply call the hook inside TemplateDetailsLeftGrid

Reply:
also, in another file u send prop editable to TemplateDetailsRightGrid , you have in line 56 the component, but not sending the prop?